db/create/create_table_pokemon.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE `pokemon` (
```

Comment:
Call the table `Pokemon`. This will also be the name of the SQL Alchemy later on.